### ISSUE TITLE
Old version of ublaboo/responses is installed when installing datagrid v5.4.11

### ISSUE THREAD

**Issue by Salamek** [ublaboo/datagrid#602]
This breaks PHP 7.2 Compability

**Comment by paveljanda** [comment]
@Salamek Can you send a PR? :)

**Comment by Salamek** [comment]
@paveljanda 
nope :disappointed: 

```bash
  Problem 1
    - ublaboo/responses v2.0.1 requires nette/application ^3.0 -> satisfiable by nette/application[3.0.x-dev, v3.0.0-alpha1] but these conflict with your requirements or minimum-stability.
    - ublaboo/responses v2.0.0 requires nette/application ^3.0 -> satisfiable by nette/application[3.0.x-dev, v3.0.0-alpha1] but these conflict with your requirements or minimum-stability.
    - Installation request for ublaboo/responses ~2.0.0 -> satisfiable by ublaboo/responses[v2.0.0, v2.0.1].
```
There is no version of ublaboo/responses targeting PHP 7.2 && nette 2.4 would be good to release v1.1.0 of ublaboo/responses targeting PHP 7.2 && nette 2.4

**Comment by paveljanda** [comment]
Fixed in `ublaboo/responses` `v1.0.7`

**Comment by Salamek** [comment]
@paveljanda THX, works great
